.gitlab-ci.yml

```diff
@@ -242,10 +256,59 @@ publish:s3:doc:
   after_script:
     - aws s3 ls s3://${BUCKET}/${PREFIX}/
         --human-readable --summarize
-  tags:
-    - linux-docker
 
 
 
 
 
+
+
+.deploy:template:                  &deploy
+  stage:                           deploy
+  when:                            manual
+  cache:                           {}
+  retry:                           1
+  image:                           dtzar/helm-kubectl:$HELM_VERSION
```

Comment:
I understand this is a microsoft developer, however, is it okay with our security policies to use a personal third-party docker-image to deploy production?

//cc @kirushik

Reply:
@gnunicorn well, actually it's a valid concern.
I don't think we have a general policy here — but maybe forking https://github.com/dtzar/helm-kubectl to paritytech, setting up DockerHub autobuilds and using the resulting image would be enough? The `Dockerfile` in that repo seems to be a low-maintenance thing.

Reply:
we now have our own: https://hub.docker.com/r/parity/kubectl-helm